src/rsbr.java

```diff
@@ -97,18 +97,18 @@ private static int numberOfEdges(int rows, int columns) {
         LinkWeightTracker lwTracker = new LinkWeightTracker(g, volumes);
         int choice = 5;
         switch (choice) {
-            case 0: // Sem seleção
+            case 0: // Sem sele��o
```

Comment:
I don't know why it states that I deleted and readded these lines, I didn't modify this part of the code.